plugins/out_clp_s3/flush/flush.go

```diff
@@ -0,0 +1,87 @@
+// Package implements method to send data to output
+package flush
+
+import (
+	"C"
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+	"unsafe"
+
+	"github.com/fluent/fluent-bit-go/output"
+
+	"github.com/y-scope/fluent-bit-clp/config"
+)
+
+// flushes data to file
```

Comment:
```suggestion
// Flushes data to file
```

Match capitals in other doc strings